src/views/v2/components/read-more.php

```diff
@@ -9,15 +9,14 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.10
- *
+ * @version TBD
  */
+
 ?>
+<span class="tribe-events-c-small-cta-hellip"> &hellip; </span>
 <div class="tribe-events-c-small-cta tribe-common-b3 tribe-events-c-small-cta--readmore">
 	<a
-		href="<?php echo esc_url( get_permalink( get_the_ID() ) ) ?>"
+		href="<?php echo esc_url( get_permalink( get_the_ID() ) ); ?>"
```

Comment:
Shouldn't this be getting the ID from the `$event` object rather than the `get_the_ID()` template tag?

Reply:
No, because we are not sending the `$event` there, given that if we get it, it would give a fatal: https://github.com/moderntribe/the-events-calendar/pull/2741#discussion_r333066555

Reply:
I see what we're hooking into from https://github.com/moderntribe/the-events-calendar/pull/2741. Instead, can we do the `get_permalink( get_the_ID() )` in the hook and pass down a `$link` variable? We want to maintain consistency with our other templates. Ideally we would want to pass down `$event` too, but seems like we can't. If users want to modify the read more template, they still have access to the template tags.

Reply:
Yes I agree on that, we should be passing down either the Event Object or at least a link.

Reply:
As we discussed before, it not possible to pass the event object here.